src/DataWrangling/JRA55.jl

```diff
@@ -252,7 +252,15 @@ function set!(fts::JRA55NetCDFFTS, path::String=fts.path, name::String=fts.name)
 
     ti = time_indices(fts)
     ti = collect(ti)
-    data = ds[name][i₁:i₂, j₁:j₂, ti]
+
+    if issorted(ti)
+        data = ds[name][i₁:i₂, j₁:j₂, ti]
+    else # ti must wrap around 1
```

Comment:
I don't understand what "wrap around 1" mean
I think I understand what you mean when I see the linked issue; but if the issue wasn't there I wouldn't know..
I presume this refers to using forcing from 1 year to force for longer than a year? shall we make the comment more informative?

Reply:
what do you suggest?

Reply:
[b7c6dcf](https://github.com/CliMA/ClimaOcean.jl/pull/287/commits/b7c6dcf2b04cbe3438700cc4661e0f2a3d145d51) is great!